runtime/wasm/src/module/mod.rs

```diff
@@ -647,7 +645,16 @@ where
                         "entity_operations" => ops.len(),
                         "time" => format!("{}ms", start_time.elapsed().as_millis())
                     );
-                    self.ctx.state.entity_operations.extend(ops);
+                    for op in ops {
```

Comment:
I think it's worth changing the variable name from `ops` to `output_states` or something that is more accurate.